packages/react-components/src/components/ChatMessage/ChatMessageActionMenu.tsx

```diff
@@ -39,16 +38,18 @@ export const chatMessageActionMenuProps = (menuProps: {
         key="menuButton"
         data-ui-id="chat-composite-message-action-icon"
         ref={menuProps.menuButtonRef}
-        onClick={() => menuProps.onActionButtonClick()}
-        style={{ margin: '1px' }}
+        onClick={showActionMenu ? () => menuProps.onActionButtonClick() : undefined}
+        style={{ margin: showActionMenu ? '1px' : 0, minHeight: showActionMenu ? undefined : '30px' }}
         role="button"
         aria-label={menuProps.ariaLabel}
```

Comment:
It looks like it will be visible for accessibility even when it's hidden because of:
- `tabIndex={0}`
- `aria-label={menuProps.ariaLabel}`

Reply:
Nice catch